docs/badge.md

```diff
@@ -73,6 +96,13 @@ import { Avatar, Badge } from 'react-native-elements'
 
 ---
 
+## Props for withBadge
```

Comment:
Let's move this down with the reference for withBadge.

So it should read:
Props
Reference
Props for withBadge
Reference for withBadge